### PR TITLE
Remove BitBucket references from Firefox documentation

### PR DESCRIPTION
**Pull request by flodolo**

### CODE REVIEW

File: src/products/firefox_desktop/adding_pontoon.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
...
-* Push at least one change to the repository, otherwise Pontoon will not pick it up during sync. The safest change it to initialize the `toolkit/chrome/global/intl.properties` file with the correct values ([example](https://bitbucket.org/mozilla-l10n/ppl/commits/b3fd0faf59b0b45b2cf30c01d85157beee2a0bd0 )).
+* If the locale is not [available](https://pontoon.mozilla.org/teams/) in Pontoon yet, [consult this document](../../tools/pontoon/adding_new_locale.md) for instructions on how to add it.
+* File a bug to create a new Mercurial repository in [l10n-central](https://hg.mozilla.org/l10n-central/). Several bug templates, including one to request a new repository, are available in [this page](https://l10n.mozilla.org/bugs/new-locale). For more information about these templates, [refer to this document](adding_nightly.md).
+* Push at least one change to the repository, otherwise Pontoon will not pick it up during sync. The safest change it to initialize the `toolkit/chrome/global/intl.properties` file with the correct values ([example](https://hg.mozilla.org/l10n-central/ppl/rev/b3fd0faf59b0b45b2cf30c01d85157beee2a0bd0)).
```
Comment (peiying2) [inline src/products/firefox_desktop/adding_pontoon.md]: Rewrite the second sentence: The safest way to change it is to ....

**Review by peiying2 — CHANGES_REQUESTED** [review]
One sentence needs to take a second look. Otherwise, it looks good.

File: src/products/firefox_desktop/adding_pontoon.md (hunk 1 above)
Comment (flodolo) [reply src/products/firefox_desktop/adding_pontoon.md]: Thanks, looks like it was already there and missed at least twice.

**Review by peiying2 — APPROVED** [review]
Much simpler change, :). LGTM!

**Comment by flodolo** [comment]
Thanks!

**Pull request merged**